src/game_io/game_preload_packet.cc

```diff
@@ -63,10 +63,12 @@ void GamePreloadPacket::read(FileSystem& fs, Game&, MapObjectLoader* const) {
 			background_ = s.get_safe_string("background");
 			// TODO(Nordfriese): Savegame compatibility
 			background_theme_ = (packet_version < 7 ? "" : s.get_safe_string("theme"));
+#if 0  // TODO(Nordfriese): Re-add training wheels code after v1.0
 			training_wheels_wanted_ =
 			   (packet_version < 8 ? false : s.get_safe_bool("training_wheels"));
 			active_training_wheel_ =
 			   (packet_version < 9 ? "" : s.get_safe_string("active_training_wheel"));
+#endif
```

Comment:
This is a version rollback now. The `kCurrentPacketVersion` should represent it. Otherwise we get problems, when re-adding this code. Alternatively the packet version could be increased again after re-adding, which you should note in the comment.